src/OpenTelemetry/Trace/Tracing.cs

```diff
@@ -44,12 +43,18 @@ internal Tracing()
                     eventQueue);
 
             tracer = new Tracer(startEndHandler, TraceConfig);
+            tracerFactory = new TracerFactory(tracer);
         }
 
+        /// <summary>   
+        /// Gets the factory for creating named Tracers.
+        /// </summary>
+        public static ITracerFactory TracerFactory => tracerFactory;
+        
         /// <summary>   
         /// Gets the tracer to record spans.
         /// </summary>
-        public static ITracer Tracer => (ITracer)tracer;
+        public static ITracer Tracer => tracer; // TODO: We could probably get rid of this property in favor of TracerFactory.
```

Comment:
Yes, I'd say we should get rid of this. However, we might still need it for compat reasons :/.

Concerning "startEndHandler, TraceConfig" properties that the global Tracer currently receives. Aren't these properties that the TracerFactory should also get?

Reply:
Agreed. Adressed in follow-up commit.